lib/pbench/server/api/resources/upload_api.py

```diff
@@ -224,7 +226,7 @@ def put(self, filename: str):
                     hash_md5.update(chunk)
             except Exception:
                 self.logger.exception(
-                    "Unexpected error uploading {}, user = {}, ctrl = {}",
+                    "Unexpected error uploading {!a}, user = {}, ctrl = {!a}",
```

Comment:
Breaking the user, ctrl, file pattern here makes a bit more sense since you're specifically calling out a problem with uploading the file... however I also kinda like the idea of having a consistent ordering through the method, and it wouldn't be too bad. You already say "Uploading file" for the pre-load INFO, so "Unexpected error uploading file" here wouldn't be out of place and putting the filename at the end would be fine.